conf/application.conf

```diff
@@ -138,6 +138,8 @@ mongo-async-driver {
 # Microservice specific config
 clientSecretLimit = 5
 upliftVerificationValidity = 90d
+unusedApplicationsDeletionCutoff = 365d
```

Comment:
I don't think we need this as these are the default values in the provider anyway.

Reply:
I think we should still remove these two properties